src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignedAssemblyOrExcuse.java

```diff
@@ -131,7 +131,7 @@ public static void writeSAMFile( final String samFile, final PipelineOptions pOp
                     final List<BwaMemAlignment> alignments = allAlignments.get(contigIdx);
                     if ( alignments.isEmpty() ) continue;
                     final FermiLiteAssembly.Contig contig = assembly.getContig(contigIdx);
-                    final String readName = String.format("asm%06d:tig%05d", assemblyId, contigIdx);
+                    final String readName = formatAssemblyID(assemblyId) + ":" + formatContigID(contigIdx); //String.format("asm%06d:tig%05d", assemblyId, contigIdx);
```

Comment:
Vastly less efficient (but called infrequently enough that it doesn't matter).  But I also find the revision less clear, to tell you the truth.